infra/README.md

```diff
@@ -202,6 +211,56 @@ tags = merge(module.common.default_tags, {
 
 })
 ```
+
+## Destroying Infrastructure
+
+To destroy everything you'll need to undo everythin in reverse.
+
+1. First destroy all your environments by running `terraform destroy` in each of your environment module folders
+
+    ```bash
+    # within each env folder in /infra/envs/ (dev, stage, etc)
+    terraform destroy
+    ```
+
+2. Then to destroy the backends, first you'll need to add `force_destroy = true` to the S3 buckets, and update the
+  lifecycle block to set `prevent_destroy = false`. Then run `terraform apply`
+
+    ```terraform
+    resource "aws_s3_bucket" "tf_state" {
+      bucket = var.state_bucket_name
+
+      force_destroy = true
+
+      # Prevent accidental destruction a developer executing terraform destory in the wrong directory. Contains terraform state files.
+      lifecycle {
+        prevent_destroy = false
+      }
+    }
+
+    ...
+
+    resource "aws_s3_bucket" "tf_log" {
+      bucket = var.tf_logging_bucket_name
+      force_destroy = true
+    }
+    ```
+
+3. Then since we're going to be destroying the tfstate buckets, you'll want to unconfigure them, so you want to comment out or delete the s3 backend configuration and run `terraform init -force-copy` to copy the tfstate back to a local tfstate file.
```

Comment:
```suggestion
3. Then since we're going to be destroying the tfstate buckets, you'll want to move the tfstate file out of S3 and back to your local system. Comment out or delete the s3 backend configuration and run `terraform init -force-copy` to copy the tfstate back to a local tfstate file.
```

Reply:
"unconfigure" sounded a little vague to me 👀

Reply:
sounds good, i was lazily using that wording based on the terraform log output when you do it:

> Terraform has detected you're unconfiguring your previously set "s3" backend.
> Successfully unset the backend "s3". Terraform will now operate locally.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/447859/193302602-4383f8bc-af16-4859-bb77-fad575343488.png">